courses/serializers/v2/programs.py

```diff
@@ -104,35 +105,75 @@ class ProgramSerializer(serializers.ModelSerializer):
     def get_courses(self, instance) -> list[int]:
         return [course[0].id for course in instance.courses if course[0].live]
 
+    def get_collections(self, instance) -> list[int]:
+        return [
+            collection.id
+            for collection in ProgramCollection.objects.filter(programs__id=instance.id)
+        ]
+
     @extend_schema_field(
```

Comment:
I'm not 100% certain that changing the structure of `requirements` here doesn't have any side effects. The unit tests are passing, but I'm particularly interested if someone with more experience in this codebase knows if this will be problematic. If so, we can change the approach.